mibs/HOST-RESOURCES-MIB.mib

```diff
@@ -0,0 +1,1540 @@
+HOST-RESOURCES-MIB DEFINITIONS ::= BEGIN
```

Comment:
You've added a lot of standard mibs that we already have, please remove these as they aren't needed.

For any specific to this OS or vendor please upload to `mibs/teleste/` dir